# 카메라 프레임에 대한 파라미터
FRAMES_PER_SECONDS: 10

LEFT_DEPTH_COLOR_TOPIC: /depth_01/color/image_raw
LEFT_DEPTH_REGISTERED_TOPIC: /depth_01/depth_registered/points

LEFT_DEPTH_INTRINSIC_VALUE: [456.1969909667969, 0.0, 327.2201232910156, 0.0, 456.1969909667969, 246.26255798339844, 0.0, 0.0, 1.0]

# 단위: meter
LEFT_ARUCO_EDGE_SIZE: 0.130
LEFT_ARUCO_GAP_SIZE: 0.043

LEFT_DEPTH_IMAGE_01_TOPIC: /depth_01_image_01
LEFT_DEPTH_IMAGE_02_TOPIC: /depth_01_image_02

LEFT_DEPTH_CAMERA_POINTCLOUD_TOPIC: /depth_01_camera_pointcloud
LEFT_DEPTH_WORLD_POINTCLOUD_TOPIC: /depth_01_world_pointcloud

LEFT_DEPTH_TF_TOPIC: depth_01_tf

RIGHT_DEPTH_COLOR_TOPIC: /depth_02/color/image_raw
RIGHT_DEPTH_REGISTERED_TOPIC: /depth_02/depth_registered/points

RIGHT_DEPTH_INTRINSIC_VALUE: [452.2761535644531, 0.0, 324.9412841796875, 0.0, 452.2761535644531, 244.85147094726562, 0.0, 0.0, 1.0]

# 단위: meter
RIGHT_ARUCO_EDGE_SIZE: 0.13
RIGHT_ARUCO_GAP_SIZE: 0.043

RIGHT_DEPTH_IMAGE_01_TOPIC: /depth_02_image_01
RIGHT_DEPTH_IMAGE_02_TOPIC: /depth_02_image_02

RIGHT_DEPTH_CAMERA_POINTCLOUD_TOPIC: /depth_02_camera_pointcloud
RIGHT_DEPTH_WORLD_POINTCLOUD_TOPIC: /depth_02_world_pointcloud

RIGHT_DEPTH_TF_TOPIC: depth_02_tf